.github/workflows/jira-pr.yaml

```diff
@@ -0,0 +1,21 @@
+on:
+  pull_request:
+   types:
+      - edited
+      - opened
+      - synchronize
+      - reopened
+
+name: Jira PR
+
+jobs:
+  validate_jira_reference:
+    runs-on: ubuntu-latest
```

Comment:
can you change to be self-hosted?

Reply:
```suggestion
    runs-on: [ARM64, self-hosted, Linux]
```

Reply:
@jakeyheath Is that the correct change?

Reply:
yup! just make sure this runs okay on ARM, but that is the correct syntax

Reply:
Sounds good, we'll see it run in a sec once I merge the change

Reply:
Seems like it works as expected

Reply:
🚢